src/bulletprooflist.js

```diff
@@ -4,51 +4,75 @@ var cheerio = require("cheerio");
 * Converts ordered and unordered lists to tables.
 */
 var bulletproofList = ((function(cheerio) {
-	function createProcessLiFn(numbered) {
-		var bullet = ((function() {
-			if (numbered) {
-				return function(idx) {
-					return idx + 1 + ".";
-				};
-			}
-
-			return function() {
-				//return "*";
-				return "&#x02022;"; //not good: Lotus Notes 7, Outlook 2013
-				//return "&#8226;"; //not good: Lotus Notes 7, Outlook 2013
-				//return "&bull;";
-				//return "&bullet;";
+    
+
+	function listElemProcessorFn(numbered) {
+        var tableAligns = [];
+        
+        var alignmentModifier = (function () {
+
+			return function (idx, actLiElem) {
+                var act = $(actLiElem);
+                var alignProp = act.css(["text-align"])["text-align"];
+                
+                if (alignProp) {
+                    tableAligns.push(alignProp);
+                }
 			};
-		})());
-
-		return function processLi(idx, actLiElem) {
-			var act = $(actLiElem);
-
-			var actContent = act.html();
-
-			var tr = $("<tr></tr>");
-
-			tr.append($("<td align=\"left\" width=\"15\" valign=\"top\">" + bullet(idx) + "</td>"));
-			tr.append($("<td align=\"left\"></td>").html(actContent));
-			act.replaceWith(tr);
-		};
-	}
-
-	function createProcessListElemFn(numbered) {
-		var liProcessor = createProcessLiFn(numbered);
-
-		return function processListElem(act) {
-			act.children("li").each(liProcessor);
-
-			var table = $("<table cellpadding=\"0\" cellspacing=\"0\" border=\"0\"></table>");
-
+		})();
+        
+        var liProcessor = (function (numbered) {
+            var bullet = ((function listElemMarker() {
+                if (numbered) {
+                    return function(idx) {
+                        return idx + 1 + ".";
+                    };
+                }
+    
+                return function() {
+                    //return "*";
+                    return "&#x02022;"; //not good: Lotus Notes 7, Outlook 2013
+                    //return "&#8226;"; //not good: Lotus Notes 7, Outlook 2013
+                    //return "&bull;";
+                    //return "&bullet;";
+                };
+            })());
+    
+            return function listElemBulletproofer(idx, actLiElem) {
+                var act = $(actLiElem);
+    
+                var actContent = act.html();
+    
+                var tr = $("<tr></tr>");
+
+                tr.append($("<td align=\"left\" width=\"15\" valign=\"top\">" + bullet(idx) + "</td>"));
+                tr.append($("<td align=\"left\"></td>").html(actContent));
+                act.replaceWith(tr);
+            };
+        })(numbered);
+
+		return function processListElems(act) {
+            
+            act.children("li").each(alignmentModifier);
+            act.children("li").each(liProcessor);
+
+            // CHECK LIST ITEM ALIGNMENT
+            var equalityChecker = tableAligns.every((v, i, a) => i === 0 || v === a[i - 1] );
```

Comment:
by the name of the variable, I would guess it's a function. Also, you use var and const... why don't you use const all the time?